package.json

```diff
@@ -6,8 +6,8 @@
   "scripts": {
     "test": "jest",
     "test:examples": "athloi run test --filter 'example-*'",
-    "build": "athloi run build --filter 'dotcom-*' --concurrency 10 --preserve-order",
-    "build:examples": "athloi run build --filter 'example-*' --concurrency 10",
+    "build": "tsc --build --verbose tsconfig.build.json && npm run build:browser --workspaces --if-present",
```

Comment:
I aimed to have a script to build all the typescript files for both our node and browser needs but wasn't able to figure that out. This builds all typescript files by default for node then we have a task that runs in each package with a `build:browser` script defined.

Reply:
**praise:** Nice! It's great to read your tech lead publicly say "I wasn't able to figure that out" and it's no big deal 😁